android/src/main/java/io/customer/reactnative/sdk/extension/MapExtensions.kt

```diff
@@ -3,7 +3,7 @@ package io.customer.reactnative.sdk.extension
 import com.facebook.react.bridge.ReadableMap
 
 internal fun ReadableMap?.toMap(): Map<String, Any> {
-    return this?.toHashMap()?.filterValues { it != null } ?: emptyMap()
+    return this?.toHashMap()?.filterValues { it != null }?.mapValues { it.value as Any } ?: emptyMap()
```

Comment:
Wouldn't it be nicer to simply check for `null` and avoid unnecessary casting? Both should work though.

```suggestion
    return this?.toHashMap()?.filterValues { it != null }?.mapValues { it.value != null } ?: emptyMap()
```